egs2/dcase_task1/asr1/local/data.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+
+stage=1
+stop_stage=100000
+log "$0 $*"
+. utils/parse_options.sh
+
+. ./db.sh
+. ./path.sh
+. ./cmd.sh
+
+DCASE="/ocean/projects/cis210027p/siddhana/TAU-urban-acoustic-scenes-2022-mobile-development/"
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+if [ -z "${DCASE}" ]; then
+    log "Fill the value of 'FSC' of db.sh"
```

Comment:
Is it really FSC?

Reply:
No this was typo, it should be DCASE. Good catch!